portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +24,31 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<String> messages;
+    @Override
+    public void init() {
+        messages = new ArrayList<String>();
+        messages.add("hello");
+        messages.add("hi");
+        messages.add("howdy");
+    }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String message = "Hello Shreya!";
-    response.setContentType("text/html;");
-    response.getWriter().println(message);
+    String json =  convertToJson(messages);
+    //send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(ArrayList<String> arr) {
+      int i;
```

Comment:
Need to either add an early return if `arr` is empty or change the logic below so that it works for an empty input.